routes/categories.py

```diff
@@ -39,8 +45,91 @@ def get_category_by_id(category_id):
         session.close()
         abort(404, "Category with id = %s not found" % category_id)
 
-    json_category = to_json(cat)
+    locale = validate_locate(request.headers.get('locale'))
+    json_category = convert_to_json(cat, locale)
 
     session.close()
 
     return json_category
+
+
+put_category_schema = {
+    'type': 'object',
+    'properties': {
+        'name': {'type': 'string'},
+    },
+    'required': ['name']
+}
+
+
+@category_blueprint.route('/categories', methods=['PUT'])
+@expects_json(put_category_schema)
+def put_new_place():
+    session = get_session()
+    content = g.data
+
+    locale = validate_locate(request.headers.get('locale'))
+
+    category = Category()
+    name_id = str(uuid4())
+    locale_string = LocaleString(
+        id=name_id,
+        locale=locale,
+        text=content['name']
+    )
+    category.name_id = name_id
+    category.name.set(locale_string)
+
+    session.add(category)
+
+    session.commit()
+    session.close()
+
+    return 'ok'
+
+
+@category_blueprint.route('/categories/<category_id>', methods=['POST'])
+@expects_json(put_category_schema)
+def post_place_by_id(category_id):
+    session = get_session()
+    category = session.query(Category).get(category_id)
+
+    if category is None:
+        session.close()
+        abort(404, "Category with id = %s not found" % category_id)
+        return
+
+    content = g.data
+    locale = validate_locate(request.headers.get('locale'))
+
+    if locale not in category.name:
```

Comment:
Try to just use category.name.set(LocaleString(...)) without this, if it will work i think it will be better to use it.